### ISSUE TITLE
Is it possible to change a font for a theme ?

### ISSUE THREAD

**Issue by cbdRedGG** [adi1090x/polybar-themes#197]
Hi, I'm using Shades theme and I tried to change the default font but It didn't work so I was wondering if it's possible to do so

Thank's for your work, those themes are truly awesome.

Greetings.

**Comment by sukalaper** [comment]
> Hi, I'm using Shades theme and I tried to change the default font but It didn't work so I was wondering if it's possible to do so
> 
> Thank's for your work, those themes are truly awesome.
> 
> Greetings.

Yes of couse. Generally, to change the font is to put it on `font-0`. And make sure your fonts are available with `fc-list grep -i "Your-font"`.
